src/profile_manager/templates/profile_manager/profile_list.html

```diff
@@ -167,7 +171,9 @@
 
                 {% if view_type != VIEW_TYPES.STAFF and view_type != VIEW_TYPES.INACTIVE %}
 
-                    <td>{{ object.grad_year|default:"-" }}</td>
+                    {% if config.custom_profile_field %}
+                    <td>{{ object.custom_profile_field|default:"-" }}</td>
```

Comment:
Consistancy with default value? above you put None here it's "-"

Reply:
fixed. Using "None"